.github/workflows/image.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - get-matrix
+    if: ${{ matrix.frameworkonly != "true" }}
```

Comment:
This wont work,  I dont think you can access the matrix at the job level :(

Reply:
yes I see the failure now :( this is unfortunate